templates/hooks/identity_provider_config/sdk_delete_post_build_request.go.tpl

```diff
@@ -0,0 +1,6 @@
+	identityProviderConfigType := "oidc"
```

Comment:
Can we make this variable a constant in the hooks.go file?

Reply:
done